tests/ci/cdk/cdk/codebuild/github_ci_integration_omnibus.yaml

```diff
@@ -250,3 +250,23 @@ batch:
         image: 620771051181.dkr.ecr.us-west-2.amazonaws.com/aws-lc-docker-images-linux-x86:ubuntu-22.04_gcc-12x_latest
         variables:
           AWS_LC_CI_TARGET: "tests/ci/integration/run_ntp_integration.sh"
+
+    - identifier: pq_tls_integration_x86_64
+      buildspec: tests/ci/codebuild/common/run_simple_target.yml
+      env:
+        type: LINUX_CONTAINER
+        privileged-mode: false
+        compute-type: BUILD_GENERAL1_SMALL
+        image: 620771051181.dkr.ecr.us-west-2.amazonaws.com/aws-lc-docker-images-linux-x86:ubuntu-22.04_gcc-12x_latest
+        variables:
+          AWS_LC_CI_TARGET: "tests/ci/integration/run_pq_tls_integration.sh"
+
+    - identifier: pq_tls_integration_aarch
```

Comment:
I'll defer to the LC team, but I don't think we need to cover both ARM and Intel for this test. This is purely application layer stuff so the CPU arch really doesn't matter.

Reply:
Agreed, we're a bit inconsistent about that for other projects but just one platform is fine.